src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -1561,6 +1579,13 @@ NetworkOPsImp::setAmendmentBlocked()
     setMode(OperatingMode::TRACKING);
```

Comment:
While we're here, this should probably change too: `TRACKING` means "convinced we agree with the network" per the documentation. If we are amendment blocked we *might* transiently agree with the network, but the fact that we are actually **UNABLE** to process transactions the network is able to generate suggests that this should be `CONNECTED` instead.

Reply:
Good point. Done.